modules/linear_boltzmann_solvers/executors/pi_keigen_scdsa.cc

```diff
@@ -254,7 +245,7 @@ PowerIterationKEigenSCDSA::Execute()
     double reactivity = (k_eff_ - 1.0) / k_eff_;
 
     // Check convergence, bookkeeping
-    k_eff_change = fabs(k_eff_ - k_eff_prev) / k_eff_;
+    k_eff_change = std::fabs(k_eff_ / k_eff_prev - 1.0);
     k_eff_prev = k_eff_;
```

Comment:
Should some of these be moved inside the verbose statement if they are only used there?

Reply:
Moved reactivity to the verbose statement.